### PR TITLE
Chart/Tooltip: refactored style declaration

### PR DESCRIPTION
**Pull request by thisisobate**
**What this PR does / why we need it**:
refactored style declaration
**Which issue(s) this PR fixes**:

Relates #30354

### CODE REVIEW

File: packages/grafana-ui/src/components/Chart/Tooltip.tsx [hunk 1]
```diff
@@ -68,3 +62,16 @@ export const Tooltip: React.FC<TooltipProps> = ({ content, position, offset }) =
 };
 
 Tooltip.displayName = 'ChartTooltip';
+
+const getStyles = (theme: GrafanaTheme) => {
+  return {
+    portal: css`
+      position: absolute;
+      top: ${theme.breakpoints.xs};
+      left: ${theme.breakpoints.xs};
```
Comment (hugohaggmark) [inline packages/grafana-ui/src/components/Chart/Tooltip.tsx:71]: Not sure about this one, maybe just use `0` in this case.
```suggestion
      top: 0;
      left: 0;
```

Comment (thisisobate) [reply packages/grafana-ui/src/components/Chart/Tooltip.tsx:71]: Alright 👍

**Review by hugohaggmark — APPROVED** [review]
Looks good, I've not have time to test it though.

**Comment by thisisobate** [comment]
@torkelo awaiting your reviews 🥳

**Review by torkelo — APPROVED** [review]
Looks good, works as before :)

**Pull request merged**